src/main/resources/com/google/api/codegen/java/PUBLISHING.md

```diff
@@ -0,0 +1,31 @@
+PUBLISHING
+----------
+
+Note: This folder has been generated by the gen-api-package tool.
+
+The instructions assume that no changes have been made to the folder and its
+contents since it was created.
+
+PREREQUISITES
+-------------
+
+- JDK 7 must be installed.
+
+TO TEST
+-------
+
+```
+  $ ./gradlew install
+```
+
+Then find the package in `~/.m2/repository/com/google/api/grpc/`
```

Comment:
This directory depends on the group name of the package, which is now hard coded in grpc_package.snip. We can parameterize this later if we feel we need to parameterize group name in general.